test/reporters.js

```diff
@@ -1116,7 +1116,6 @@ describe('Reporter', () => {
                 expect(result.lint[0].filename).to.exist();
                 expect(result.lint[0].errors).to.exist();
                 done();
-                done = function () {};
```

Comment:
It's been done on purpose by https://github.com/hapijs/lab/commit/3e14d48960b5ca69e7a5f4c3e482090af7f37a9e. @geek was that a mistake ?